core/jvm/src/main/scala/zio/sql/table.scala

```diff
@@ -135,8 +134,8 @@ trait TableModule { self: ExprModule with SelectModule =>
     trait Insanity {
       def ahhhhhhhhhhhhh[A]: A
     }
-    sealed trait Source extends Table with Insanity {
-      type Repr[_]
+    // NOTE: Moving Repr[_] from a type parameter to a type member breaks deleteFrom compilation on 2.12
+    sealed trait Source[Repr[_]] extends Table with Insanity {
```

Comment:
can you please remove the insanity trait and see if this change has fixed the "//The outer reference in this type test cannot be checked at run time?!" comments in the rendering?

Reply:
@robmwalsh the error is still present (not sure if relevant, changing `Source` to `sealed abstract class` makes the warning go away).

Reply:
Great, thanks for looking into that I don't think there's a reason Source needs to be a trait vs sealed abstract class, so might look to do that and remove `Insanity`. Will await John's verdict on this PR and take it from there.